cfgmgr/portmgr.cpp

```diff
@@ -76,6 +77,50 @@ bool PortMgr::setPortAdminStatus(const string &alias, const bool up)
     return true;
 }
 
+
+bool PortMgr::setPortDHCPMitigationRate(const string &alias, const string &dhcp_rate_limit)
+{
+    stringstream cmd;
+    string res, cmd_str;
+    int ret;
+    int byte_rate = stoi(dhcp_rate_limit) * 406;
+
+    if (dhcp_rate_limit != "0")
+    {
+        // tc qdisc add dev <port_name> handle ffff: ingress
+        // &&
+        // tc filter add dev <port_name> protocol ip parent ffff: prio 1 u32 match ip protocol 17 0xff match ip dport 67 0xffff police rate <byte_rate>bps burst <byte_rate>b conform-exceed drop
+        cmd << TC_CMD << " qdisc add dev " << shellquote(alias) << " handle ffff: ingress" << " && " \
+            << TC_CMD << " filter add dev " << shellquote(alias) << " protocol ip parent ffff: prio 1 u32 match ip protocol 17 0xff match ip dport 67 0xffff police rate " << to_string(byte_rate) << "bps burst " << to_string(byte_rate) << "b conform-exceed drop";
+        cmd_str = cmd.str();
+        ret = swss::exec(cmd_str, res);
+        SWSS_LOG_WARN("ret Value in setPortDHCPMitigationRate is  ret:%d,", ret);
```

Comment:
Why is this log warn? Seems it doesn't mean unexpected situation?

Reply:
the log SWSS_LOG_WARN("ret Value in setPortDHCPMitigationRate is ret:%d,", ret); was added temporarily for debugging purposes to track failures and will be removed.

Reply:
Please remove this warning.